Doc/make.bat

```diff
@@ -180,7 +180,9 @@ if EXIST "%BUILDDIR%\html\index.html" (
 goto end
 
 :check
-cmd /S /C "%SPHINXLINT% -i tools"
+rem --severity=0 enables all checks, but we don't want to check for long lines
+cmd /S /C "%SPHINXLINT% -i tools --severity=0 --disable='line too long'"
+cmd /S /C "%SPHINXLINT% --severity=0 --disable='line too long' ..\Misc\NEWS.d\next\ "
```

Comment:
If I make these changes then the `.bat` script picks up the two true-positive lint errors in exactly the same way as the makefile

```suggestion
cmd /S /C "%SPHINXLINT% -i tools --severity=0 --disable=^"line too long^""
cmd /S /C "%SPHINXLINT% --severity=0 --disable=^"line too long^" ..\Misc\NEWS.d\next\ "
```

Reply:
Thanks for testing and for the fix!

Apparently there are [a few ways of escaping quotes in bat files](https://stackoverflow.com/a/31413730): `^"`, `\"`, `""`, and `"""`.  If `\"`/`""` also work, I have a slight preference for those, since they are more commonly used and understood than `^"`.

Reply:
Agreed with Alex, `^"` is unfortunately the way to go.  Except when it isn't, but here it probably is.

Escaping in `.bat` is a nightmare.